incubator/command-config/src/main/java/io/aklivity/zilla/runtime/command/config/internal/openapi/OpenApiHttpProxyConfigGenerator.java

```diff
@@ -0,0 +1,366 @@
+{
+    private static final JsonPatch ENV_VARS_PATCH = createEnvVarsPatch();
+
+    private final OpenApi openApi;
+    private final ConfigWriter configWriter;
+
+    public OpenApiHttpProxyConfigGenerator(
+        Path input)
+    {
+        this.openApi = parseOpenApi(input);
+        this.configWriter = new ConfigWriter(null);
+    }
+
+    public String generateConfig()
+    {
+        return writeConfig(createNamespaceConfig());
+    }
+
+    private OpenApi parseOpenApi(
+        Path input)
+    {
+        OpenApi openApi = null;
+        try (InputStream inputStream = new FileInputStream(input.toFile()))
+        {
+            Jsonb jsonb = JsonbBuilder.create();
+            openApi = jsonb.fromJson(inputStream, OpenApi.class);
+            jsonb.close();
+        }
+        catch (Exception ex)
+        {
+            ex.printStackTrace();
+            rethrowUnchecked(ex);
+        }
+        return openApi;
+    }
+
+    private NamespaceConfig createNamespaceConfig()
+    {
+        NamespaceConfigBuilder<NamespaceConfig> namespaceBuilder = NamespaceConfig.builder()
+            .name("example");
+
+        Map<String, GuardedConfig> guardedRoutes = new HashMap<>();
+        for (String securitySchemeName : openApi.components.securitySchemes.keySet())
+        {
+            String guardType = openApi.components.securitySchemes.get(securitySchemeName).bearerFormat;
+            if ("jwt".equals(guardType))
+            {
+                namespaceBuilder.guard()
+                    .name("jwt0")
+                    .type(guardType)
+                    .options(JwtOptionsConfig::builder)
+                        .issuer("") // env
+                        .audience("") // env
+                        .key()
+                            .alg("").kty("").kid("").use("").n("").e("").crv("").x("").y("") // env
+                            .build()
+                        .build()
+                    .build();
+                GuardedConfig guarded = GuardedConfig.builder().name("jwt0").role("echo:stream").build();
+                guardedRoutes.put(securitySchemeName, guarded);
+            }
+        }
+
+        BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>> httpServer0BindingBuilder = namespaceBuilder
+            .binding()
+                .name("tcp_server0")
+                .type("tcp")
+                .kind(SERVER)
+                .options(TcpOptionsConfig::builder)
+                    .host("0.0.0.0")
+                    .ports(resolvePortsForScheme("https"))
+                    .build()
+                .route()
+                    .exit("tls_server0")
+                    .build()
+                .build()
+            .binding()
+                .name("tcp_server1")
+                .type("tcp")
+                .kind(SERVER)
+                .options(TcpOptionsConfig::builder)
+                    .host("0.0.0.0")
+                    .ports(resolvePortsForScheme("http"))
```

Comment:
This feels as though we are pulling the OpenAPI specification into the zilla.yaml instead of generating zilla.yaml based on the contents of the OpenAPI specification.

For example, what if there are no `http` scheme URLs in the OpenAPI specification (e.g. they are all `https`)?